feature_engine/selection/drop_psi_features.py

```diff
@@ -475,7 +475,7 @@ def fit(self, X: pd.DataFrame, y: pd.Series = None):
                 threshold_cat = self.threshold
 
         # Compute the PSI by looping over the features
-        self.psi_values_ = {}
```

Comment:
I'm totally not against type hints. But to be consistent with the style of the script, perhaps is better not to add it (or to add it everywhere ;-) )

Reply:
but typehint checks gives an errors because of this line

Reply:
Ok interesting. Are tests failing for that? Because at line 479 you have `self.features_to_drop_ = []` Why is good in this case?

Reply:
I can see you've done it already, sorry.